src/test/java/net/juniper/netconf/NetconfSessionTest.java

```diff
@@ -136,7 +136,7 @@ public void GIVEN_executeRPC_WHEN_lldpRequest_THEN_correctResponse() throws Exce
 
         Thread thread = new Thread(() -> {
             try {
-                outPipe.write("Go to HELL".getBytes());
```

Comment:
Lol

Reply:
hahaha

Reply:
Sorry guys, my bad. Forgot to change it :(